UCM test: different results when re-run in same workspace

To reproduce:
1. Edit the test `test_ucm_regression_factors` so that `args['workspace_dir']` is not temporary: `args['workspace_dir'] = '/Users/emily/Documents/ucm-test2'`
2. Run the UCM tests (all pass)
3. Run the UCM tests again (now `test_ucm_regression_factors` fails with an error: `AssertionError: 10560639.842782581 != 3520213.280928277 within 4 places (7040426.561854305 difference) : avd_eng_cn should be close to 3520213.280928, actual: 10560639.842783`

You should be able to re-run any model in the same workspace and get the same result, so it's concerning that the aggregate results are different.

--- a/HISTORY.rst
+++ b/HISTORY.rst
@@ -38,6 +38,9 @@ Unreleased Changes
 ------------------
 * General
     * Update to FontAwesome 5 icons in the QT interface.
+    * Updating the ``taskgraph`` requirement to ``0.11.0`` to resolve an issue
+      where modifying a file within a roughly 2-second window would fool
+      ``taskgraph`` into believing that the file had not been modified.
 * Habitat Quality
     * Changed how Habitat Rarity outputs are calculated to be less confusing.
       Values now represent a 0 to 1 index where before there could be

--- a/requirements.txt
+++ b/requirements.txt
@@ -20,7 +20,8 @@ Rtree>=0.8.2,!=0.9.1
 Shapely>=1.7.1,<2.0.0
 scipy>=1.6.0  # pip-only
 pygeoprocessing>=2.3.2  # pip-only
-taskgraph[niced_processes]>=0.10.3  # pip-only
+#taskgraph[niced_processes]>=0.10.3  # pip-only
+git+https://github.com/phargogh/taskgraph.git@bugfix/76-flaky-zerodivisionerror-on-linux-tests#egg=taskgraph
 psutil>=5.6.6
 chardet>=3.0.4
 openpyxl

--- a/requirements.txt
+++ b/requirements.txt
@@ -20,8 +20,7 @@ Rtree>=0.8.2,!=0.9.1
 Shapely>=1.7.1,<2.0.0
 scipy>=1.6.0  # pip-only
 pygeoprocessing>=2.3.2  # pip-only
-#taskgraph[niced_processes]>=0.10.3  # pip-only
-git+https://github.com/phargogh/taskgraph.git@bugfix/76-flaky-zerodivisionerror-on-linux-tests#egg=taskgraph
+taskgraph[niced_processes]>=0.11.0  # pip-only
 psutil>=5.6.6
 chardet>=3.0.4
 openpyxl

